lib/logstash/outputs/loggly.rb

```diff
@@ -59,6 +59,12 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   # https://www.loggly.com/docs/source-groups/
   config :tag, :validate => :string, :default => "logstash"
 
+  # Retry count
+  config :retry_count, :validate => :number, :default => 5
+
+  # Can Retry
+  config :can_retry, :validate => :boolean, :default => false
```

Comment:
thanks @jordansissel , I am thinking to keep the default value to be 5 so that user can get the leverage of the retries